### DISCUSSION TITLE
Étude de cas : "Analyse du système d'information 2024" P24

### DISCUSSION THREAD

**Discussion by louisduhalberruer** [Hypertopic/HyperGlosae#215] [Show and tell]
Here is the video og the features we added to Hyperglosae.

@nitram35 @raphaelweis @petitfa1, @Barnabepetit

https://github.com/Hypertopic/HyperGlosae/assets/19203540/f1c7f59d-20f0-4dcf-afa1-39df57ce4f49

**Comment by benel** [comment]
@raphaelweis @louisduhalberruer @nitram35 @Barnabepetit @petitfa1
Je vous transmets le retour d'Hugo Jeanningros : 

> Merci pour le travail fourni, qui fait avancer Hyperglosae afin que nous puissions un jour réaliser IF14 par ce biais. Les fonctionnalités implémentées correspondent bien à ce qui avait été commandé. Vous trouverez ci-dessous quelques commentaires relatifs à de futures améliorations possibles :
>
> 0:39 : Une enquête sociologique n’est pas équivalente à une enquête policière et en IF14 on ne s’occupe pas de la seconde. Attention donc, nous ne sommes pas dans la même démarche. « Enquête » veut simplement dire récolter des données de terrain et les analyser.
>
> 1:50 : Attention sur les boutons cliquables, il faut éviter d’implémenter un bouton «  call to action » dans un autre.
>
> 3:40 : pour le feedback d’erreur relatif au droit de modification, une petite fenêtre pop up avec un message clair (ex: vous ne disposez pas des droits de modifications) serait plus adaptée que l’ouverture d’une fenêtre avec l’indication technique de l’erreur.
>
> 4:05 : Il y a beaucoup d’icônes autour des titres de documents, les utilisateurs pourraient s’y perdre. Il faut faire attention à les rendre explicites, à leur homogénéisation et à leur alignement.
>
> 4:19 : il n’y a pas de feedback utilisateur lorsque l’on tape le nom pendant l’ajout des droits pour un tiers.
